re-frame/test/martian/re_frame_test.cljs

```diff
@@ -104,7 +102,7 @@
 
      (re-frame/reg-fx
       ::martian/request
-      (fn [[_ operation-id & args]]
+      (fn [[_ _ operation-id & args]]
```

Comment:
Will need to note that this is a breaking change to anyone who has overridden this effect (e.g. in their own tests)